coops-ui/src/views/Dashboard.vue

```diff
@@ -245,7 +245,8 @@ export default {
      */
     checkPendingFilings (filings) {
       filings.forEach(filing => {
-        if (filing.status === FilingStatus.PAID) {
+        if (this.entityFilter(EntityTypes.BCORP) &&
+          filing.status === FilingStatus.PAID) {
           this.effectiveDate = filing.filingEffectiveDate
           this.coaPending = true
```

Comment:
Since the coaPending is set here should we check whether the filing type is change of address?

A filing can be in a paid status if the filer is down for some reason. It can be for any type of filing. In that scenario I am not sure whether caution icon will show up next to another filing type.

Reply:
You're 100% right, i will add that validation in as well and update the tests. Without there is a potential that a BCOMPs Annual report or Director change could trigger this state. Thanks!